### DISCUSSION TITLE
Organization error

### DISCUSSION THREAD

**Discussion by MarJose123** [community/community#61491] [Mobile] [answered]
**Select Topic Area**
Bug

**Body**

I've created an Organization with a space on its Organization Display name in the GitHub web, and accessing it from GitHub mobile it returns a message "Invalid user or Organization name" it should be accessible in the mobile since the organization name url is in slug format.

**Comment by D3vil0p3r** [comment]
I have the same issue

**Reply by ThnksCJ** [reply]
I have also encountered this issue

**Reply by aakashsyadav1999** [reply]
I am experience same issue

**Reply by aconsuegra** [reply]
Please, update to version 1.123 and give it another try. Thanks!

**Comment by aconsuegra** [comment, marked as answer]
This was a valid bug introduced in the Android app in the previous version. It is now fixed in the public Beta version of the app (1.124), and in the latest Production build (1.123) which is being rolled out gradually this week.
